trollmoves/server.py

```diff
@@ -192,7 +213,7 @@ def push(self, message):
         new_msg = self._move_files(message)
         if new_msg is None:
             new_msg = Message(message.subject,
-                              message.type,
+                              'file',
```

Comment:
Don't we have `dataset` types of messages coming here?

Reply:
Could be? I don't remember seeing those in my own setups. If you have such a message, I could perhaps adapt this. Or would it be sufficient to check the message data for `'uri'`, `'dataset'` and `'collection'` and decide based on that? The incomming message in any case has type `'push'`.

Reply:
Oh, I guess @TAlonglong had one example in an issue, I'll have a look after lunch.

Reply:
Ok, done.